controllers/component_build_controller_pac.go

```diff
@@ -902,3 +1164,62 @@ func retrievePipelineSpec(bundleUri, pipelineName string) (*tektonapi.PipelineSp
 	pipelineSpec := pipelineSpecObj.PipelineSpec()
 	return &pipelineSpec, nil
 }
+
+// upddateIncoming updates incomings in repository, adds new incoming for provided branch with incoming secret
```

Comment:
typo `upddate`
Also, could you be more specific what is / how / when updated, like:
If there is no incomings, one incoming is added.
If there are two or more, they are merged into one.
...

Reply:
ok